module.json

```diff
@@ -57,7 +61,7 @@
     "manifest": "https://github.com/League-of-Foundry-Developers/compendium-browser/releases/latest/download/module.json",
     "download": "https://github.com/League-of-Foundry-Developers/compendium-browser/releases/download/v0.7.2/compendium-browser.zip",
     "minimumCoreVersion": "0.8.6",

```

Comment:
Can I suggest that until this version is confirmed to work in v10, you change the `minimumCoreVersion` to 10? Just in case your v10 changes break the (currently working version) for Foundry 0.8.